src/helpers/_focusable.scss

```diff
@@ -10,10 +10,18 @@
 ///
 /// @access public
 
+$govuk-focus-width-inner: 2px;
+
 @mixin govuk-focusable {
   &:focus {
     outline: $govuk-focus-width solid $govuk-focus-colour;
     outline-offset: 0;
+    box-shadow: inset 0 0 0 $govuk-focus-width-inner $govuk-text-colour;
+
+    @include govuk-if-ie8 {
```

Comment:
This feels too specific to the input components, and would likely break anything else.

Perhaps IE8 could use user-agent styles instead?

Reply:
Radios and checkboxes set the width for their styles so it'd be a bit weird to not do it on inputs.